test/test_company_ja.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'helper'
+
+class TestCompanyJA < Test::Unit::TestCase
+  class << self
+    def startup
+      FFaker::CompanyJA.const_set(:LAST_NAMES, %w[佐藤 高橋 佐々木])
+    end
+
+    def shutdown
+      FFaker::CompanyJA.class_eval { remove_const(:LAST_NAMES) }
+    end
```

Comment:
`remove_const`はもともとの状況と同じじゃないと思います。

多分‥
```suggestion
    def startup
      @original_last_names = FFaker::CompanyJA::LAST_NAMES
      FFaker::CompanyJA.const_set(:LAST_NAMES, %w[佐藤 高橋 佐々木])
    end

    def shutdown
      FFaker::CompanyJA.const_set(:LAST_NAMES, @original_last_names)
    end
```

Reply:
途中でpushしたためCIが失敗していましたがこの辺は多分関係なかったと思います…
とはいえ、ここまで細かいテストを書くのは面倒だったので `AddressJA`と同じようにかんたんなregexのみで見るように変えました。